library/src/main/java/com/dslplatform/json/DslJson.java

```diff
@@ -469,10 +469,15 @@ protected JsonReader initialValue() {
 		this.doublePrecision = settings.doublePrecision;
 		this.maxNumberDigits = settings.maxNumberDigits;
 		this.maxStringSize = settings.maxStringBuffer;
-		this.writerFactories.addAll(settings.writerFactories);
-		this.readerFactories.addAll(settings.readerFactories);
-		this.binderFactories.addAll(settings.binderFactories);
 		this.classLoaders.addAll(settings.classLoaders);
+		ExternalConverterAnalyzer externalConverterAnalyzer = new ExternalConverterAnalyzer();
```

Comment:
We might wanna add a settings on DslJson.Settings to allow such external lookups (it's fine as is for now)